src/api/EpiData.ts

```diff
@@ -150,13 +150,17 @@ export function loadDataSet(
     });
 }
 
-export function fetchCOVIDcastMeta(): Promise<{ geo_type: string; signal: string; data_source: string }[]> {
+export function fetchCOVIDcastMeta(): Promise<
+  { geo_type: string; signal: string; data_source: string; time_type?: string }[]
```

Comment:
Was this filled in by your IDE because you use the `.time_type` member accessed in `COVIDcast.svelte`?

Why is the `time_type` marked "optional" with the question mark modifier?  Is that just inherited from the `importCOVIDcast()` call signature definition?  Or is it because the argument has a default value for `time_type`?

Reply:
> Was this filled in by your IDE because you use the .time_type member accessed in COVIDcast.svelte?

yes!

> Was this filled in by your IDE because you use the .time_type member accessed in COVIDcast.svelte? Why is the time_type marked "optional" with the question mark modifier? Is that just inherited from the importCOVIDcast() call signature definition? Or is it because the argument has a default value for time_type?

Just inherited from that call.